src/styles.css

```diff
@@ -1 +1,5 @@
 /* You can add global styles to this file, and also import other style files */
+ "styles": [
```

Comment:
And don't forget to install it https://github.com/inspirehep/record-editor/blob/master/package.json#L52